test/errors-spec.js

```diff
@@ -61,16 +61,15 @@ beforeEach(function() {
 describe("Parser:", function() {
 
     describe("#handleInfixNodes", function() {
-        // TODO: The position information here is broken, should be fixed.
         it("rejects repeated infix operators", function() {
             expect("1\\over 2\\over 3").toFailWithParseError(
-                   "only one infix operator per group at position -1: " +
-                   "1\\over 2\\over ");
+                   "only one infix operator per group at position 9: " +
+                   "1\\over 2\\̲o̲v̲e̲r̲ 3");
```

Comment:
The key is not just having a single position, but a range formed by a pair of positions. With that, underlining each character is a piece of cake. Although the actual rendering may show huge differences in quality depending on fonts or font rendering engines.

It *might* be possible to [install some special code](http://bit.ly/object-formatters) to pretty-print KaTeX error messages in an even nicer way for Chrome…